inst/include/dust/dust.hpp

```diff
@@ -4,18 +4,25 @@
 #include <dust/rng.hpp>
 
 #include <algorithm>
+#include <map>
```

Comment:
As far as I can tell, we always access individual elements of the `map`, and don't iterate through it. So I think we should be using `unordered_map`.